.werft/workspace-run-integration-tests.sh

```diff
@@ -15,6 +15,8 @@ DO_CLEANUP=0
 declare -A FAILURE_TESTS
 declare SIGNAL # used to record signal caught by trap
 
+BRANCH_TIMEOUT_SEC=10800
```

Comment:
I'd prefer we extend it to 1 day to prevent a case that the integration job runs over 3 hours.
If others trigger the integration test, it will clean up the branch.

Reply:
@jenting Thanks for your review :bow: The timeout of the job is 2 hours, so we don't need to wait for 1day. WDYT?

Reply:
Could you please check those two? It looks like we hit timeout 😅 
https://werft.gitpod-dev.com/job/gitpod-workspace-run-integration-tests-main.127
https://werft.gitpod-dev.com/job/gitpod-workspace-run-integration-tests-main.128

Reply:
@jenting Currently all tests flow normally just in time for the werft timeout. In other words, if one test case eats time until its timeout of the unit test, it will reach the werft timeout. So this is natural.
Let's create an issue about the test that is failing.